src/ProjectTemplates/test/template-baselines.json

```diff
@@ -938,6 +938,26 @@
       ]
     }
   },
+  "signalr": {
+    "None": {
+      "Template": "signalr",
+      "Arguments": "new signalr",
+      "Files": [
+        "Hubs/ChatHub.cs",
+        "Hubs/StreamingTimerHub.cs",
```

Comment:
This line should probably be removed now. I do like the the idea of adding a streaming method to ChatHub if we can find a way to do it that makes sense.

Reply:
Maybe we could just subscribe to a stream of messages instead of relying on the Send callback. I'd like to demonstrate how to define client methods too though. So then we'd need to add a callback for something else. Maybe we can have specific callbacks for people joining and leaving the chat.

Reply:
what say you, @halter73, about me just adding the streaming time method to the chat hub? whilst it isn't relevant to chat, it'd cover the base.

Reply:
Yeah. Maybe we could leave the "Send" client hub method as is and use a streaming method to update server time every second or something.